JS/js1.js

```diff
@@ -0,0 +1,18 @@
+// // FORM SUBMIT
+
+let forms = document.querySelectorAll('form')
```

Comment:
I see you have only one `<form>` tag in the HTML. 
This means you can use `document.querySelector` instead, and you don't need to loop over the forms in line 5